dist/obs-server.spec

```diff
@@ -484,7 +484,7 @@ for i in production.rb ; do
 done
 SECRET_KEY="/srv/www/obs/api/config/secret.key"
 if [ ! -e "$SECRET_KEY" ]; then
-  ( umask 0077; dd if=/dev/urandom bs=256 count=1 2>/dev/null |sha256sum| cut -d\  -f 1 >$SECRET_KEY )
+  RAILS_ENV=production bundle exec rake secret > $SECRET_KEY
```

Comment:
ups ... this opens a security hole ... why did you change that?

Also loading rails on package installation feels a bit too much. We would also need preinstallation dependencies to get this working in complex scenarios.